src/UI/EmailSignupHTMLTooltop.js

```diff
@@ -0,0 +1,44 @@
+import HTMLTooltip from './HTMLTooltip.js'
+
+class EmailSignupHTMLTooltip extends HTMLTooltip {
+    /**
+     * @param {import("../DeviceInterface/InterfacePrototype").default} device
+     */
+    render (device) {
+        this.device = device
+
+        this.shadow.innerHTML = `
+${this.options.css}
+<div class="wrapper wrapper--email">
+    <div class="tooltip tooltip--email tooltip--email-signup" hidden>
+        <h1>
+            Protect your inbox 💪 I've caught trackers hiding in 85% of emails.
+        </h1>
+        <p>
+            Want me to hide your email address and remove hidden trackers before
+            forwarding messages to your inbox?
+        </p>
+        <div class="notice-controls">
+            <a href="https://duckduckgo.com/email/start-incontext" target="_blank" class="primary">
+                Get Email Protection
+            </a>
+            <button class="ghost js-dismiss-email-signup">
```

Comment:
Lol, why the class `ghost`? Is this specified in the design language?

Reply:
@GioSensation Yea, "ghost" is usually the name for this kind of button, where the background only shows on hover/focus (I'll need to check I've implemented that). You can see the ghost button react component on the SERP storybook